src/App.tsx

```diff
@@ -1,61 +1,137 @@
 import './App.scss';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
you store it `boolean` value, not a string
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
```